src/java/fr/paris/lutece/portal/web/system/CacheJspBean.java

```diff
@@ -138,6 +143,24 @@ public String doReloadProperties( HttpServletRequest request ) throws AccessDeni
         return JSP_MANAGE_CACHES;
     }
 
+    /**
+     * Reload the logs configuration of the application
+     *
+     * @return The URL to display when the process is done.
+     * @throws AccessDeniedException
+     */
+    public String doReloadLogsConfiguration( HttpServletRequest request ) throws AccessDeniedException
+    {
+        if ( !SecurityTokenService.getInstance( ).validate( request, TEMPLATE_MANAGE_CACHES ) )
+        {
+            throw new AccessDeniedException( "Invalid security token" );
+        }
+        AppLogService.init( PATH_CONF, FILE_PROPERTIES_CONFIG );
+
+        return JSP_MANAGE_CACHES;
+    }
+

```

Comment:
You should be able to add tests that

- ensure the CSRF protection is enforced
- verify that the log configuration is indeed changed